string type1
string info1
string type2
string info2
string type3
string info3
---
string phrase
